relayer/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please remove this file

Reply:
My bad, removed [here](https://github.com/Snowfork/polkadot-ethereum/pull/413/commits/57b896d214ba8d81a53fb20a342952c511af97ea)